test/libsolidity/ASTJSON/abstract_contract_parseOnly.json

```diff
@@ -0,0 +1,57 @@
+{
+  "absolutePath": "a",
+  "exportedSymbols": {},
+  "id": 6,
+  "nodeType": "SourceUnit",
+  "nodes":
+  [
+    {
+      "abstract": true,
+      "baseContracts": [],
+      "contractDependencies": [],
+      "contractKind": "contract",
+      "fullyImplemented": true,
```

Comment:
I think it's totally fine that some members are missing, but I'm worried that wrong information might be very misleading.